### PR TITLE
Add user agent string to api client

### PR DESCRIPTION
**Pull request by solidgoldpig**
### What changed

Add user agent string to api client

### Why did it change

This enables us to distinguish our client requests from others using the same library

### Issue tracking

n/a

## Screenshots

n/a

#### Automated testing

- [x] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics

### CODE REVIEW

**Comment by merlinc** [comment]
I think this is a good idea. 

My only question is how does the` api` layer want us to supply this sort of data? I did q quick search through the `api` codebase and didn't see anything relevant about `user-agent`.

**Review by merlinc — COMMENTED** [review]
I think this is a good idea. 

My only question is how does the` api` layer want us to supply this sort of data? I did q quick search through the `api` codebase and didn't see anything relevant about `user-agent`.

**Comment by solidgoldpig** [comment]
> I think this is a good idea.
> 
> My only question is how does the` api` layer want us to supply this sort of data? I did q quick search through the `api` codebase and didn't see anything relevant about `user-agent`.

It's a halfway-house quick fix. Rather than figuring out whether a particular `axios` request if from the frontend app or not, it will be self-evident. Obviously this does not go the whole hog and tie the api request back to a parent request by a user... but that's for laters.

File: common/lib/api-client/request-headers.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ const uuid = require('uuid')
 
 const {
   API: { VERSION },
+  APP_VERSION,
 } = require('../../../config')
 
 module.exports = (format = 'application/vnd.api+json') => {
   return {
+    'User-Agent': `hmpps-book-secure-move/${APP_VERSION}`,
```
Comment (teneightfive) [inline common/lib/api-client/request-headers.js:10]: Do we want to append `-fronted` to be more explicit: `hmpps-book-secure-move-frontend/${APP_VERSION}`?

**Review by teneightfive — APPROVED** [review]

File: common/lib/api-client/request-headers.js (hunk 1 above)
Comment (solidgoldpig) [reply common/lib/api-client/request-headers.js]: Done

Comment (merlinc) [reply common/lib/api-client/request-headers.js]: @teneightfive Good idea, that makes it more obvious it is from our `frontend` client

**Pull request merged**